tensorflow_serving/example/hot_reload_grpc_example.py

```diff
@@ -0,0 +1,38 @@
+#coding=utf-8
+from tensorflow_serving.apis import model_service_pb2
+from tensorflow_serving.apis import model_service_pb2_grpc
+from tensorflow_serving.apis import model_management_pb2
+from tensorflow_serving.config import model_server_config_pb2
+from tensorflow_serving.util import status_pb2
+
+import grpc
+
+def run():
+  channel = grpc.insecure_channel('yourip:yourport')
+  stub = model_service_pb2_grpc.ModelServiceStub(channel)
+  request = model_management_pb2.ReloadConfigRequest()  ##message ReloadConfigRequest
+  model_server_config = model_server_config_pb2.ModelServerConfig()
+
+  config_list = model_server_config_pb2.ModelConfigList()##message ModelConfigList
+  
+  one_config = config_list.config.add() #####try to add one model config
+  one_config.name= "test1"
+  one_config.base_path = "/home/model/model1"
+  one_config.model_platform="tensorflow"
+
+  model_server_config.model_config_list.CopyFrom(config_list) c
```

Comment:
c is not required here

Reply:
1.it seems that the serving supports this feature from r1.7？
2.you should pass the complete model list as a model_config_list pb

you can find more information in my github: https://github.com/sparkingarthur/serving/tree/master/custom_scripts

Reply:
How can I set model_version_policy to 'all' or 'specific' ?